search-app/src/cmr/search/api/autocomplete.clj

```diff
@@ -0,0 +1,77 @@
+(ns cmr.search.api.autocomplete
+  "Defines the API for autocomplete-suggest in the CMR."
+  (:require
+    [clojure.string :as string]
+    [compojure.core :refer :all]
+    [cmr.common.log :refer (debug info warn error)]
+    [cmr.common.mime-types :as mt]
+    [cmr.common.services.errors :as svc-errors]
+    [cmr.common-app.api.routes :as common-routes]
+    [cmr.search.api.core :as core-api]
+    [cmr.search.services.autocomplete-service :as ac]
+    [cmr.search.services.query-service :as qs]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Support Functions
+
+(def page-size-default 20)
+(def page-num-default 0)
+
+(defn- string-param-or-default->int
+  [s default]
+  (if s
+    (Integer/parseInt s 10)
```

Comment:
Remove `10` from the call. It is confusing to have it here.

Reply:
Default page size is defined in `cmr.common-app.services.search.query-model`. As well as `default-offset` and `default-page-num`. Although i thought you would get this for free? Take a look at `parse-query-level-params`. I think you should add another multimethod for `:autocompelete`. Unless you're not able to do that?

Reply:
i wrong the above comment in the wrong spot but i think its still understandable.

Reply:
As long as not declaring the radix is ok, I will remove it